svn_select_to_git.py

```diff
@@ -814,13 +815,22 @@ def FindTreeOrphans(dir1, dir2):
   os.chdir(dir1)
   for root, dirs, files in os.walk("."):
     if (root[0:len(groot)] != groot):
-      for file in files:
-        fname1 = os.path.join(root, file).lstrip("./")
-        fname2 = os.path.join(dir2, fname1)
-        if (not os.path.exists(os.path.join(dir2, fname1))):
-          orphans.append(fname1)
-        # End if
-      # End for
+      #Check if directory is an external:
+      ext_check = external_cam_check(root)
+      #Add file to orphan list if not an external:
+      if not ext_check:
```

Comment:
I think you can get rid of this check if you add `"--ignore-externals"` to the svn export command (line 343).

Reply:
I am not sure this works, as "manage_externals" is an SVN external, and so is not downloaded with this flag.  I could add additional code to manually grab just the "manage_externals" external, but that would just result in replacing the "ext_check" code with additional code somewhere else.  The other option is we could add "manage_externals" as a git subtree or submodule, but since it wasn't clear what the decision on that was, I decided to leave it as-is.  Anyways, please let me know if you have an opinion one way or the other.  Thanks!

Reply:
Oh details :)
I thought we were just going to copy in manage_externals into every commit. Hmm, how much work is that?

- Need a function to check if manage_externals remote is added
- Need function to add manage_externals remote

```
git remote add -f --tags manage_externals https://github.com/ESMCI/manage_externals
```
- Need a function to add manage_externals to the dir
```
git read-tree --prefix=<external_subdir>/ -u a48558d890d46c51c2508f97aed64b5dd1716b74
```
Sound doable?